### ISSUE TITLE
The Blazor will normalize the aria-selected='true' to aria-selected, so can we fix the code add this condition check?

### ISSUE THREAD

**Issue by rwecho** [themesberg/flowbite#745]
**Describe the bug**
I developed a Blazor app with Flowbite. Despite the Tabs components working well, the Blazor will normalize the 'aria-selected' attribute without a 'true' value.  The final result of this issue is that the default selected tab does not work.

**To Reproduce**
Create a Blazor project, referencing the Flowbite and tabs component. Then create the tab item in a loop logic.

**Expected behavior**
The default selected tab works in the Blazor app.

**Screenshots**
<img width="984" alt="image" src="https://github.com/themesberg/flowbite/assets/8048753/6122f62f-c597-4817-b8f7-379f538fafed">

**Desktop (please complete the following information):**
All platforms supported by Blazor

**Smartphone (please complete the following information):**
All devices supported by Blazor
